documentation/assemblies/upgrading/assembly-upgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='assembly-upgrade-cluster-operator-{context}']
+= Upgrading the Cluster Operator
+
+[role="_abstract"]
+How you upgrade the Cluster Operator depends on the way you deployed it.
+
+Using installation files:: If you deployed the Cluster Operator using the installation YAML files, perform your upgrade by modifying the Operator installation files, as described in xref:proc-upgrading-the-co-{context}[Upgrading the Cluster Operator].
+
+Using the {OperatorHub}:: If you deployed Strimzi from {OperatorHub}, use the Operator Lifecycle Manager (OLM) to change the update channel for the Strimzi operators to a new Strimzi version.
++
+Depending on your chosen upgrade strategy, after updating the channel, either:
++
+** An automatic upgrade is initiated
++
+** A manual upgrade will require approval before the installation begins
++
+For more information on using OperatorHub.io to upgrade Operators, see the {OLMOperatorDocs}.
+
+Using a Helm chart:: If you deployed the Cluster Operator using a Helm chart, use `helm upgrade`.
++
+The `helm upgrade` command does not upgrade the {HelmCustomResourceDefinitions}.
+Install the new CRDs manually after upgrading the Cluster Operator.
+You can access the CRDs from the {ReleaseDownload} or find them in the `crd` subdirectory inside the Helm Chart.
+
+[id='con-upgrade-cluster-operator-unsupported-kafka-{context}']
+== Upgrading the Cluster Operator returns Kafka version error
+
+If you apply a multi-version upgrade to a Cluster Operator that currently uses an unsupported Kafka version, you get an _unsupported Kafka version_ error.
```

Comment:
I don't think it is clear what _multi-version upgrade_ is.

```suggestion
If you upgrade the Cluster Operator while having a deployed Kafka cluster with older Kafka version which is not supported by the new operator version anymore, you get an _unsupported Kafka version_ error.
```

Reply:
Changed to:

> If you upgrade the Cluster Operator and get an _unsupported Kafka version_ error, your Kafka cluster deployment has an older Kafka version that is not supported by the new operator version.